src/app/_components/PageHeader.tsx

```diff
@@ -71,12 +75,17 @@ PageHeader.Image = function PageHeaderImage({
   image,
 }: Pick<PageHeaderProps, 'image'>) {
   const isStaticImage = 'data' in image
+
   const commonProps = {
     alt: image.alt,
     priority: true,
     quality: 100,
     sizes: buildImageSizeProp({ startSize: '100vw', lg: '490px' }),
-    className: 'rounded-lg border border-brand-100',
+    className: clsx(
+      'rounded-lg border border-brand-100',
+      image.objectFit === 'cover' && 'object-cover',
+      image.objectFit === 'contain' && 'object-contain',
```

Comment:
Should we have it default to `cover`?

Reply:
In the spirit of our Image refactoring, I made everything explicit so that we know what is going on :)